src/components/Chart.vue

```diff
@@ -28,7 +37,16 @@ export default {
     const history = lastSnapshot.date !== todaySnapshot.date
       ? [...this.history, todaySnapshot]
       : this.history
-    this.chart = new Chart(ctx, {
+
+    this.$refs.alertDayAndPriceRef.style.opacity = '0'
+    this.$refs.alertDayAndPriceRef.style.position = 'absolute'
```

Comment:
this should probably be css, it would be great to get some guidance on where to add this logic in the repo 👍